packages/gestalt/src/Table.css

```diff
@@ -7,40 +7,54 @@
 
 .th {
   composes: alignStart from "./Typography.css";
-  composes: paddingY3 from "./boxWhitespace.css";
-  composes: paddingX3 from "./boxWhitespace.css";
 }
 
+.th,
 .td {
   composes: paddingY3 from "./boxWhitespace.css";
   composes: paddingX3 from "./boxWhitespace.css";
 }
 
+.stickyHeader {
+  box-shadow: var(--elevation-raised-top);
```

Comment:
We're already declaring these styles as `.raisedTop` and `.raisedBottom` in Borders.css. Let's compose these styles from those classes to keep bundle sizes down.

Reply:
DONE